### DISCUSSION TITLE
Adding AWS cert to stackstorm-ha pods

### DISCUSSION THREAD

**Discussion by bertyah** [StackStorm/st2#5682] [Operations]
I'm deploying Stackstorm-ha in AWS and trying to use Amazon DocumentDB instead of the mongo from the cluster. To connect to this, AWS required the download and use of one of their certificates. I created a configmap of the certificate and added it in the extra_volumes section for each pod. However the Jobs: section does not have an extra_volumes option so the register-content piece fails.

How can I get the AWS cert into the pods to be able to connect to the database?

**Comment by arm4b** [comment]
From the past reports, Amazon DocumentDB is not a fully MongoDB-compatible replacement and people have seen issues in such an environment.
So not sure if that will work.
